test/e2e/app-dir/rewrite-headers/rewrite-headers.test.ts

```diff
@@ -304,7 +304,8 @@ const cases: {
     name: 'middleware rewrite external HTML',
     pathname: '/hello/vercel',
     expected: {
-      'x-nextjs-rewritten-path': null,
+      // Vercel matches `/` to `/home`
+      'x-nextjs-rewritten-path': '/home',
```

Comment:
vercel.com now matches `/home`. I don't know why this is surfaced in our API. Feels like this is leaking implementation details of the other Next.js page we're redirecting to. 

But if it's a bug, it was always there and only surfaced after vercel.com started matching `/` to `/home` internally.

Maybe this should test against `x-middleware-rewrite` instead?